spec-main/yarn.lock

```diff
@@ -193,6 +193,11 @@ nise@^4.0.1:
     just-extend "^4.0.2"
     path-to-regexp "^1.7.0"
 
+node-addon-api@^3.0.0:
```

Comment:
This is a fairly simple module; could we use the C API of napi directly rather than adding a dependency?

Reply:
yes, i actually started off with the c api before finding out there is c++ extensions.